subsys/bluetooth/host/conn.c

```diff
@@ -1525,7 +1525,6 @@ static void deferred_work(struct k_work *work)
 
 		if (conn->type == BT_CONN_TYPE_ISO) {
 			bt_iso_disconnected(conn);
-			bt_conn_unref(conn);
```

Comment:
Would it make sense to add a code comment here so no one thinks there’s a missing unref?